src/Features/Blockcore.Features.NodeHost/NodeHostSettings.cs

```diff
@@ -40,6 +40,11 @@ public class NodeHostSettings
         /// </summary>
         public bool EnableAPI { get; private set; }
 
+        /// <summary>
+        /// If true will require authentication on all sensitive APIs. Some APIs will be public available.
+        /// </summary>
+        public bool EnableAuth { get; private set; }
```

Comment:
ah pretty cool ok so this is off by default

Reply:
I wonder if we should try to distinguish localhost and  as well

Reply:
No reason to distinguish localhost.